lib/platformos_check/language_server/completion_providers/object_completion_provider.rb

```diff
@@ -34,6 +38,19 @@ def object_to_completion(object)
           **doc_hash(content)
         }
       end
+
+      def find_file(context)
+        @storage.platformos_app
+          .grouped_files
+          .values
+          .map(&:values)
+          .flatten
+          .find { |app_file| app_file.relative_path.to_s == context.relative_path }
```

Comment:
```suggestion
        @storage.platformos_app[context.relative_path]
```

uhm, how about this ^ ?

Reply:
see:

```    def [](name_or_relative_path)
      case name_or_relative_path
      when Pathname
        all.find { |t| t.relative_path == name_or_relative_path }
      else
        all.find { |t| t.relative_path.to_s == name_or_relative_path }
      end
    end
```
we might want to memorize `@grouped_files` though inside `app.rb`